SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -120,10 +121,10 @@ public IEnumerable<string> GetActiveRuleKeys(string qualityProfile, string langu
             var properties = JArray.Parse(contents);
             var result = properties.ToDictionary(p => p["key"].ToString(), p => p["value"].ToString());
 
-            // http://jira.sonarsource.com/browse/SONAR-5891
+            // http://jira.sonarsource.com/browse/SONAR-5891 
             if (!result.ContainsKey("sonar.cs.msbuild.testProjectPattern"))
             {
-                result["sonar.cs.msbuild.testProjectPattern"] = ".*test.*";
+                result["sonar.cs.msbuild.testProjectPattern"] = SonarProperties.DefaultTestProjectPattern;
```

Comment:
The C# plugin needs to be updated with the new default value to use too, doesn't it? Please update the ticket with the new regex and check with @dbolkensteyn.